src/mscorlib/src/System/OperatingSystem.cs

```diff
@@ -138,6 +138,9 @@ private OperatingSystem()
                     case PlatformID.MacOSX:
                         os = "Mac OS X ";
```

Comment:
MacOSX is rumored to be a certified UNIX.. so why not merge it with `PlatformID.Unix`?